libs/npm/core/src/components/grid-table/GridTable.tsx

```diff
@@ -287,50 +301,50 @@ export const GridTable = <T extends object>({
   }, [initialSortBy, instance, onChangeSort, sortBy]);
 
   React.useEffect(() => {
-    if (!!initSelectedRowIds) {
+    const initRowIds = Object.keys(
```

Comment:
@CelloLiu There a bugs with the multi-select introduced when you worked on the arrow key navigation bug.  We need to test more.